tox.ini

```diff
@@ -14,8 +14,9 @@ passenv =
     PYTEST_ADDOPTS
 commands =
     python -m coverage run -m pytest {posargs}
-    python -m coverage report -m --fail-under 97
-    python -m coverage html --show-contexts
+    # Skipping py37 because coverage reports fewer statements than other versions
+    # https://github.com/nedbat/coveragepy/issues/1524
+    !py37: python -m coverage report --skip-covered --show-missing --fail-under 97
```

Comment:
Ah, right. The `!py37` doesn't work in GHA because we run `tox -e py`, and rely on the version installed by setup-python.

https://github.com/pypa/twine/blob/6659ce971b4db8003de36b803191ba3a823659fe/.github/workflows/main.yml#L59-L60

I'm gonna have to come back to this later. Might just change the threshold to 96.